src/main/kotlin/app/revanced/patcher/PatcherResult.kt

```diff
@@ -1,16 +1,32 @@
 package app.revanced.patcher
 
-import app.revanced.patcher.util.dex.DexFile
-import java.io.File
+import app.revanced.patcher.apk.Apk
 
 /**
  * The result of a patcher.
- * @param dexFiles The patched dex files.
- * @param doNotCompress List of relative paths to files to exclude from compressing.
- * @param resourceFile File containing resources that need to be extracted into the APK.
+ * @param apkFiles The patched [Apk] files.
  */
-data class PatcherResult(
-    val dexFiles: List<DexFile>,
-    val doNotCompress: List<String>? = null,
-    val resourceFile: File?
-)
\ No newline at end of file
+data class PatcherResult(val apkFiles: List<Patch>) {
+
+    /**
+     * The result of a patch.
+     *
+     * @param apk The patched [Apk] file.
+     */
+    sealed class Patch(val apk: Apk) {
```

Comment:
Yeah, that doesn't make sense. This is not the result of a single `Patch`, this is the finalized APK "bundle" the patcher came up with. Please adjust this naming accordingly, such as `PatchedApk`.

Reply:
That naming suggests that PatchedApk is Apk which isn't correct, please suggest another name

Reply:
@Sculas do you have a better name?